tests/mongo-tools/export_import_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mongotools
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestExportImport(t *testing.T) {
+	t.Parallel()
+
+	containerSourceRoot := "/dumps/mongodb-sample-databases/"
+	containerTestsRoot := "/dumps/mongodump_tests"
+	localTestsRoot := filepath.Join("..", "..", "dumps", "mongodump_tests")
+
+	type testCase struct {
+		coll           string // target collection
+		db             string // target database
+		documentsCount int    // document count
+	}
+
+	for name, tc := range map[string]testCase{
+		"Shipwrecks": {
+			coll:           "shipwrecks",
+			db:             "sample_geospatial",
+			documentsCount: 11095,
+		},
+		"Accounts": {
+			coll:           "accounts",
+			db:             "sample_analytics",
+			documentsCount: 1746,
+		},
+	} {
+		name, tc := name, tc
+		t.Run(name, func(t *testing.T) {
+			dbName1 := databaseName(t) + "_dump1"
+			dbName2 := databaseName(t) + "_dump2"
```

Comment:
```suggestion
			dbName1 := tc.db + "_export1"
			dbName2 := tc.db + "_export2"
```

When I worked on previous tests, I found testname-based database names more confusing

Reply:
I would suggest to use database and collection names here. This will create the seperate database for every test case, so it will be possible to run parallel tests on 2 collections from the same "dump"
```go
dbName1 := fmt.Sprintf("%s_%s_export1", tc.db, tc.coll)
dbName2 := fmt.Sprintf("%s_%s_export2", tc.db, tc.coll)
```